active/security.md

```diff
@@ -0,0 +1,91 @@
+# Security Team
+
+## Scope of responsibilities
+
+The security team is responsible for [Django’s security policies](https://docs.djangoproject.com/en/dev/internals/security/). This includes:
+
+- Reviewing security reports via security@djangoproject.com
+- Evaluating and patching confirmed security issues
+- Communicating with reporters
+- Communicating with the public about security releases
+- Communicating with operating-system vendors and other distributors of Django
+
+## Initial membership
+
+- Chair:
+- Co-Chair:
+- Report triagers: 
+- Steering Council Liaison (must be an active Steering Council member; may be the same as Chair/Co-Chair): Carlton Gibson
+- Board President (read-only access to mailing list): Thibaud Colas
+- Other members:
+  - Adam Johnson
+  - Carlton Gibson
+  - Jacob Walls
+  - Jake Howard
+  - James Bennett
+  - Mariusz Felisiak
+  - Markus Holtermann
+  - Michael Manfre
+  - Natalia Bidart
+  - Paul McMillan
+  - Sarah Boyce
+  - Shai Berger
+  - Simon Charette
+ 
+### Role definitions
+
+- Chair: Responsible for managing the team and seeing that responsibilities are being met.
+- Co-Chair: Responsible for helping manage the team and seeing that responsibilities are being met.
+- Report triagers: These team members are responsible for acknowleding and triaging reports initially to determine liklihood of security concern and severity.
+ 
+## Future membership
+
+The team does not have a fixed size. The team decides when new members are needed. New members are chosen from a list of volunteers. If there are no qualified volunteers the team will place an advertisement on the Django website.
+
+Members must opt-in to remain on the team on an annual basis. They may also leave for any reason.
+
+Members can also be removed by:
+
+- Becoming disqualified by the Code of Conduct working group
+- A vote of the Steering Council
+- The full consensus of the rest of the Security Team
+
+### Membership requirements
+
+Members should possess some  knowledge of the following topics, but not necessarily all of them.
+
+- Building Django applications
+- Contributing to Django
+- Web applications
+- Web security
+- Software security
+
+### How to join
+
+Any person can volunteer to join the security team by opening a PR to this repository, adding themselves to the team list. The team/WG will vote (50%+1) to approve/deny new members; the team/WG will directly vote on new Chair/Co-Chairs.
```

Comment:
Does the group want this or to ask people to email the WG or contact a member to discuss membership? 

I can imagine a flood of PRs that may not be wanted or needed.

Reply:
I switched this to a form, and indicated the details still need to be sorted out.